schemas/authenticate-hawk-response.yml

```diff
@@ -0,0 +1,45 @@
+$schema:  http://json-schema.org/draft-04/schema#
+title:                    "Hawk Signature Authentication Response"
+description: |
+  Response from a request to authenticate a hawk request.
+anyOf:
```

Comment:
it does have status now...
`status: "auth-success"` or `status: "auth-failed"`

I just use enums to enforce a specific value, depending on the object form (ie. the anyOf branch)...